### ISSUE TITLE
Imports fail for account data and streaming history

### ISSUE THREAD

**Issue by javier-cr** [Yooooomi/your_spotify#329]
Hi! A big thank you for developing this project. I am new to it and am super impressed by what you have built.

**Describe the bug**

When I try importing account data or streaming history, the import usually fails. One file has succeeded (less than 1MB), but the other ones (~10.5MB and 1MB) have all failed.

I've enabled debug logging on the container, but don't see any related errors get logged when the import fails. However, in the browser I see:

Error is:

<img width="1512" alt="Screenshot 2024-01-03 at 12 07 35 PM" src="https://github.com/Yooooomi/your_spotify/assets/44209276/e4fd811f-4a43-4e60-becb-0b44771230a6">

**Expected behavior**

The upload would normally succeed.

**Additional context**

Just to note, I am using LinuxServer's container version of this project (latest version 1.7.3).

**Screenshots**

(above)

**Shared link**

**Comment by Yooooomi** [comment]
Hello,
You might encounter a Payload too large issue. Their nginx does not support over 10mo files. Please file an issue on their repo as I think it is not related with my implementation.

**Comment by Serpensin** [comment]
> Hello, You might encounter a Payload too large issue. Their nginx does not support over 10mo files. Please file an issue on their repo as I think it is not related with my implementation.

I have the same issue with your compose file. I can't import anything (account data), no matter, how big the file is.
It tells me, that "Everything looks fine for the import to work", but I then only get 400 codes.
I copied the whole part as HAR, which I put on [pastebin](https://pastebin.com/dfUsUvqv), because I don't wanted to spam the comments here.

**Comment by KotletJB** [comment]
the same problem code 400

**Comment by KotletJB** [comment]
![image](https://github.com/Yooooomi/your_spotify/assets/62251729/59d48ae5-099e-48b2-8aff-e52f5700d55f)

**Comment by javier-cr** [comment]
Hi @Yooooomi A while back I switched over to the native your_spotify images (versus the linuxserver ones I had been using). I did still see the issue using the these images as well.

However I've recently tried this again in v.1.10.1 and everything now seems to be OK (i.e. the uploads now complete successfully), which is awesome to see. I did notice that the upload progress indicator does not update unless the page is refreshed, but that's unrelated :)

At least for my particular issue, I think this is resolved. Thank you!
